### PR TITLE
Fix missing cancel button for phone and video request details

### PR DESCRIPTION
**Pull request by jenniemc**
## Summary

This PR:
- Fix missing cancel request button for phone and video on request details page. 
- Fix request appts not to show "After visit summary" in details page. 
- Fix all request appts to only show caledar_today icon in details page. 
- Fix past and cancelled claim exam appt to show "Scheduling facility"
- Fix past and cancelled claim exam appt not to show "How to Prepare"

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#84707

## Testing done

unit test

### Request details - phone
Screenshot 2024-06-07 at 12 50 42 AM

### Request details - video
Screenshot 2024-06-07 at 12 50 23 AM
---
### Claim exam - upcoming
Screenshot 2024-06-07 at 2 41 19 PM

### Claim exam - past
Screenshot 2024-06-07 at 12 49 17 AM

### Claim exam - cancelled
Screenshot 2024-06-07 at 12 48 47 AM

## What areas of the site does it impact?

vaos

## Acceptance criteria

- [ ] Cancel request button should appear on phone and video request details page
- [ ] Phone request should show the calendar_today icon in the details page
- [ ] All requests should not display "After visit summary" in the details page
- [ ] Past and cancelled Claim Exam should not display "How to Prepare for this exam" in details page
- [ ] Past and cancelled Claim Exam should display "Scheduling Facility"

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/vaos/components/tests/VARequestLayout.unit.spec.js [hunk 1]
```diff
@@ -56,6 +58,18 @@ describe('VAOS Component: VARequestLayout', () => {
...
+            start: moment(startDate)
+              .subtract(3, 'days')
+              .format('YYYY-MM-DDTHH:mm:ss[Z]'),
+          },
+          {
+            start: moment(startDate)
+              .subtract(4, 'days')
+              .format('YYYY-MM-DDTHH:mm:ss[Z]'),
+          },
+        ],
```
Comment (jenniemc) [inline src/applications/vaos/components/tests/VARequestLayout.unit.spec.js:72]: Adding a requested date that's in the past to ensure cancel request button still displays

**Review by CBonade — DISMISSED** [review]

File: src/applications/vaos/components/tests/ClaimExamLayout.unit.spec.js
Comment (vbahinwillit) [inline src/applications/vaos/components/tests/ClaimExamLayout.unit.spec.js:1]: I would add a test to insure the schedule facility heading is not displayed.

**Review by vbahinwillit — DISMISSED** [review]
LGTM! Just added some suggestions for some additional tests cases. Thanks for finishing this one.

File: src/applications/vaos/components/layout/ClaimExamLayout.jsx [hunk 2]
```diff
@@ -104,17 +107,47 @@ export default function ClaimExamLayout({ data: appointment }) {
...
-            <span>Phone:</span>
+            {!!facility && (
+              <>
+                <br />
+                {facilityPhone && (
+                  <FacilityPhone heading="Phone:" contact={facilityPhone} />
+                )}
+                {!facilityPhone && <>Not available</>}
+              </>
+            )}
```
Comment (jenniemc) [inline src/applications/vaos/components/layout/ClaimExamLayout.jsx:150]: Updated text and included facility phone contact number

**Review by vbahinwillit — APPROVED** [review]
LGTM!

**Review by vbahinwillit — APPROVED** [review]

**Review by vbahinwillit — APPROVED** [review]

**Pull request merged**